pcb/rules/G1_7.py

```diff
@@ -14,9 +14,9 @@ def check(self):
 
         # Only perform this check on linux systems (i.e. Travis)
         # Windows automatically checks out with CR+LF line endings
-        if 'linux' in platform.platform().lower() and not self.module.unix_line_endings:
+        if 'linux' in platform.platform().lower() and not checkLineEndings(self.module.filename):
             self.error("Incorrect line endings")
-            self.errorExtra("Library files must use Unix-style line endings (LF)")
+            self.errorExtra("Footprint files must use Unix-style line endings (LF)")
```

Comment:
It is correct, but this wording is more specific. I figured that the schlib rule should differentiate between the lib and dcm files, and extended that to the footprint rule too.

Reply:
The problem is that the wording now does not agree with the website. Which is kind of where the rules are defined. The website of course does not care if the rules are checked separately for symbols and footprints. (And to be honest it might even be a good idea long term to combine the checks for generic rules where possible.)